cabal-testsuite/PackageTests/NewConfigure/setup.test.hs

```diff
@@ -0,0 +1,4 @@
+import Test.Cabal.Prelude
+
+main = setupAndCabalTest $
+    setup "configure" []
```

Comment:
I'm not sure what the point of this test script is; you're trying to test `new-configure`, not regular configure? Also, this test is redundant because we already have tests for configure; no point in making the test suite run longer! (If you delete this, make sure you delete the `out` files for this test as well.)